ts/protocol/comms.ts

```diff
@@ -71,14 +71,14 @@ export function createMessageListener<T extends MessageType>(
       console.debug('non openyolo message received');
       return false;
     }
-    if (ev.data.type !== type) return false;
+    if (ev.data['type'] !== type) return false;
```

Comment:
For these parts where messaging between the two libraries is involves, I'm using this bracket notations to make sure it's not renamed in the minification phase. I will revisit that once we have E2E tests that ensure that the two minified libraries communicate properly.